.devcontainer/Dockerfile

```diff
@@ -8,7 +8,7 @@ WORKDIR /app
 # add a non-privileged user for installing and running the application
 # We use --non-unique in case $groupid/$userid collide with the existing "vscode" user.
```

Comment:
for consistency with https://github.com/mozilla-services/tecken/pull/2882#discussion_r1465623670
```suggestion
# We use --non-unique in case $groupid/$userid collide with the existing "vscode" user.
# We use --shell /bin/bash because the default is /bin/sh which doesn't support arrow keys
```